tools/cldr-apps/src/test/resources/org/unicode/cldr/unittest/web/data/TestSTFactory.xml

```diff
@@ -202,4 +202,29 @@
 	<vote name="OrgStatM1" locale="fr_CA" xpath="//ldml/annotations/annotation[@cp='🔫'][@type='tts']">pistolet à eau</vote>
 	<vote name="OrgStatG1" locale="fr_CA" xpath="//ldml/annotations/annotation[@cp='🔫'][@type='tts']">↑↑↑</vote>
 	<verify status="approved" orgStatus="ok" statusOrg="google" locale="fr_CA" xpath="//ldml/annotations/annotation[@cp='🔫'][@type='tts']">pistolet à eau</verify>
+
+	<echo>Tests involving 20-vote items</echo>
+
+	<!-- Rules:
+	       1. For a 20 vote item, 8 votes are needed to change from non-approved to approved.
+	       2. Once Approved, it requires 20 votes to change.
+	     Note: this test assumes and requires that the .../minimumGroupingDigits path
+	     is "contributed" in fr.xml, and that the .../decimal path is "approved" in fr.xml.
+	     Ideally this test might mock the data to avoid the assumption.
```

Comment:
it might be good to have such a mock.

We already have the sandbox locales ( see CLDR-12002 ) so there's precedent for having multiple input dirs.  Actually, a `mul` sublocale such as `mul-AA` might make sense. it could have static test data, and get copied into the sandbox directory on startup for test use.